ipn/ipnserver/server.go

```diff
@@ -668,9 +673,12 @@ func StateStore(path string, logf logger.Logf) (ipn.StateStore, error) {
 	if path == "" {
 		return &ipn.MemoryStore{}, nil
 	}
+	const ephemeralPrefix = "ephemeral:"
```

Comment:
how about just making this be "mem:" ?

Reply:
I like `mem:`. The flag's about where things are stored.

Ephemeral is just an optimization that falls out naturally from that. (the fact that we then know to aggressively GC those)

`mem` is also much shorter.

And it leaves it open for us to adding an explicit `--type={,regular,ephemeral}` flag somewhere later too, orthogonal to whether storage was in memory or on disk.